.github/workflows/actions.yml

```
@@ -97,6 +97,12 @@ jobs:
       java-version: '8'
       millargs: mill.scalalib.scalafmt.ScalafmtModule/checkFormatAll __.sources
 
+  scalafix-check:
+    uses: ./.github/workflows/run-mill-action.yml
+    with:
+      java-version: '8'
+      millargs: -i __.fix && git diff --exit-code
```

I think scalafix has a built-in check feature

```suggestion
      millargs: -i -k __.fix --check
```

---

looks good